tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/contact_call_to_action.html

```diff
@@ -19,6 +18,11 @@
                 <span class="contact-cta__role">{{ value.person.role }}</span>
             {% endif %}
         </cite>
-
     {% endif %}
+
+    {% for cta in value.call_to_action %}
```

Comment:
Do you know why there is a forloop here? You can only add one button to a cta.

Reply:
Hmm i'm not sure to be honest. @lparsons396 or @engineervix - can you shed any light?

Reply:
Yeah, quite right. That's my mistake from my initial set-up of the file

Reply:
No probs - let's ticket it and resolve in another MR so not to hold testing up